api/notd/manager.py

```diff
@@ -55,49 +58,60 @@ def __init__(self, blockProcessor: BlockProcessor, saver: Saver, retriever: Retr
             ],
-            orders=[Order(fieldName=BlocksTable.c.blockDate.key, direction=Direction.DESCENDING)],
+            orders=[Order(fieldName=BlocksTable.c.blockDate.key,
+                          direction=Direction.DESCENDING)],
             limit=1
         )
         return SponsoredToken(date=baseSponsoredToken.date, token=baseSponsoredToken.token, latestTransfer=latestTransfers[0] if len(latestTransfers) > 0 else None)
 
     async def retrieve_highest_priced_transfer(self, startDate: datetime.datetime, endDate: datetime.datetime) -> TokenTransfer:
         highestPricedTokenTransfers = await self.retriever.list_token_transfers(
-            fieldFilters=[DateFieldFilter(fieldName=BlocksTable.c.blockDate.key, gte=startDate, lt=endDate)],
-            orders=[Order(fieldName=TokenTransfersTable.c.value.key, direction=Direction.DESCENDING)],
+            fieldFilters=[DateFieldFilter(
+                fieldName=BlocksTable.c.blockDate.key, gte=startDate, lt=endDate)],
+            orders=[Order(fieldName=TokenTransfersTable.c.value.key,
+                          direction=Direction.DESCENDING)],
             limit=1
         )
         return highestPricedTokenTransfers[0]
 
     async def retrieve_random_transfer(self, startDate: datetime.datetime, endDate: datetime.datetime) -> TokenTransfer:
         # NOTE(krishan711): this is no longer actually random, it's just the latest (random is too slow)
         randomTokenTransfers = await self.retriever.list_token_transfers(
-            fieldFilters=[DateFieldFilter(fieldName=BlocksTable.c.blockDate.key, gte=startDate, lt=endDate)],
-            orders=[Order(fieldName=BlocksTable.c.blockDate.key, direction=Direction.DESCENDING)],
+            fieldFilters=[DateFieldFilter(
+                fieldName=BlocksTable.c.blockDate.key, gte=startDate, lt=endDate)],
+            orders=[Order(fieldName=BlocksTable.c.blockDate.key,
+                          direction=Direction.DESCENDING)],
             offset=random.randint(1000, 10000),
             limit=1,
         )
         return randomTokenTransfers[0]
 
-    async def get_transfer_count(self, startDate: datetime.datetime, endDate: datetime.datetime) ->int:
+    async def get_transfer_count(self, startDate: datetime.datetime, endDate: datetime.datetime) -> int:
         return await self.retriever.get_transaction_count(startDate=startDate, endDate=endDate)
 
     async def retrieve_most_traded_token_transfer(self, startDate: datetime.datetime, endDate: datetime.datetime) -> TokenTransfer:
         mostTradedToken = await self.retriever.get_most_traded_token(startDate=startDate, endDate=endDate)
         query = (
-            sqlalchemy.select([TokenTransfersTable, BlocksTable]).join(BlocksTable, BlocksTable.c.blockNumber == TokenTransfersTable.c.blockNumber)
+            sqlalchemy.select([TokenTransfersTable, BlocksTable]).join(
+                BlocksTable, BlocksTable.c.blockNumber == TokenTransfersTable.c.blockNumber)
```

Comment:
undo formatting change. if you want to put stuff on new lines, move the whole function not just the params